### PR TITLE
As/add aave sl

### PR DESCRIPTION
**Pull request by adamskrodzki**
https://app.shortcut.com/oazo-apps/story/7474/deploy-aave-stop-loss-on-mainnet-and-update-common-library

AaveStopLoss added to common/Automation 

conversion of triggersData to json added to package

currently published as
@oasisdex/automation@1.3.0

### CODE REVIEW

File: packages/automation/src/mapping.ts [hunk 1]
```diff
@@ -5,6 +5,51 @@ import {
...
+    'amount',
+    'interval',
+    'recipient',
+  ],
+  [CommandContractType.AaveStopLossCommand]: [
+    'positionAddress',
+    'triggerType',
+    'collateralToken',
+    'debtToken',
+    'ltv',
```
Comment (halaprix) [inline packages/automation/src/mapping.ts:48]:
```suggestion
    'slLevel',
```

File: packages/automation/test/abi-coding.test.ts [hunk 2]
```diff
@@ -115,4 +121,102 @@ describe('abi-coding', () => {
...
+          network,
+          data,
+        );
+        expect(actual).to.deep.eq({
+          cdpId: '12',
+          triggerType: '1',
+          execCollRatio: '101',
+          maxBaseFeeInGwei: '100',
+          maxBuyPrice: '2000',
+          continous: 'true',
```
Comment (halaprix) [inline packages/automation/test/abi-coding.test.ts:173]:
```suggestion
          continuous: 'true',
```

File: packages/automation/test/abi-coding.test.ts [hunk 3]
```diff
@@ -115,4 +121,102 @@ describe('abi-coding', () => {
...
+      });
+      it('decodeTriggerDataByTypeAsJson converts to correct json', () => {
+        const actual = decodeTriggerDataByTypeAsJson(CommandContractType.BasicBuyCommand, data);
+        expect(actual).to.deep.eq({
+          cdpId: '12',
+          triggerType: '1',
+          execCollRatio: '101',
+          maxBaseFeeInGwei: '100',
+          maxBuyPrice: '2000',
+          continous: 'true',
```
Comment (halaprix) [inline packages/automation/test/abi-coding.test.ts:186]:
```suggestion
          continuous: 'true',
```

**Review by halaprix — CHANGES_REQUESTED** [review]
tests are failing - suggested changes fix it

File: packages/automation/src/mapping.ts (hunk 1 above)
Comment (adamskrodzki) [reply packages/automation/src/mapping.ts:48]: that's name for FE and it do not need to match SC, it was requested by Seba

**Review by halaprix — APPROVED** [review]

**Pull request merged**